modules/dicom_archive/help/dicom_archive.md

```diff
@@ -1,17 +1,9 @@
 # Dicom Archive
 
-The Dicom Archive module displays all scans that have been uploaded
-to LORIS, whether or not they have been inserted or not (due to protocol
-violations, mislabeling, not having been run, etc.) It's a filterable archive
-of every single dicom that this LORIS instance knows about.
+This module displays all scans that have been uploaded to LORIS. This includes scans that have been inserted, as well as those that haven’t (due to protocol violations, mislabeling, not having been run, etc.) In other words, it's an archive of every single DICOM that your LORIS instance knows about.
 
-The PatientName, and PatientID are only displayed if they've been properly
-anonymized.
+Use the *Selection Filter* section to narrow down the results in the table. Note: PatientName and PatientID are only displayed if they've been properly anonymized.
 
-Clicking on the "View Details" link in the Metadata column will take you
-to a page displaying more details of every scan and clicking on "View Images"
-in the MRI Browser column will take you to the session containing those
-images in the LORIS imaging browser.
+In the resulting table, you might notice some clickable cells. In the *Metadata* column, click **View Details** to see the compressed package of DICOM files for that scan, along with their metadata. In the *MRI Browser* column, click on **View Images** to access the session—in the LORIS imaging browser—containing those images.
```

Comment:
change to: `**View Images** to visualize and QC these scans in the Imaging Browser module`

Reply:
thanks, done

Reply:
👍